pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -144,6 +144,9 @@ self: super: builtins.intersectAttrs super {
   gtk = disableHardening (addPkgconfigDepend (addBuildTool super.gtk self.gtk2hs-buildtools) pkgs.gtk2) ["fortify"];
   gtksourceview2 = addPkgconfigDepend super.gtksourceview2 pkgs.gtk2;
   gtk-traymanager = addPkgconfigDepend super.gtk-traymanager pkgs.gtk3;
+  gi-gdkx11 = addPkgconfigDepend super.gi-gdkx11 pkgs.gtk3;
+  gi-dbusmenu = super.gi-dbusmenu.override { dbusmenu-glib = pkgs.libdbusmenu-glib; };
+  gi-dbusmenugtk3 = super.gi-dbusmenugtk3.override { dbusmenu-gtk3 = pkgs.libdbusmenu-gtk3; };
```

Comment:
These two overrides should be unnecessary after https://github.com/NixOS/cabal2nix/commit/d8bc043806a499542c4e9087ed98ba335761fde1.

Reply:
I think https://github.com/NixOS/cabal2nix/pull/353 is also needed, unless upstream is an even better place.